mapper-processor/src/main/java/com/datastax/oss/driver/internal/mapper/processor/util/generation/GeneratedCodePatterns.java

```diff
@@ -162,7 +168,25 @@ public static void setValue(
       String targetName,
       MethodSpec.Builder methodBuilder,
       BindableHandlingSharedCode enclosingClass) {
+    // invoke with SET_TO_NULL to preserve default behaviour when retrieving an entity
+    setValue(
+        cqlName,
+        type,
+        valueExtractor,
+        targetName,
+        methodBuilder,
+        enclosingClass,
+        CodeBlock.of("$T.$L", NullSavingStrategy.class, NullSavingStrategy.SET_TO_NULL));
```

Comment:
This method is using for retrieval of the entity (`@Select`) and `@Delete` - it does not apply to save so I fall back to default - `NullSavingStrategy.SET_TO_NULL` - it means that it just set nulls on the retrieved entity.
Other solution will be to create `CodeBlock.of("noOp")` and add `else if` branch here:
https://github.com/datastax/java-driver/pull/1241/files#diff-8992e8a0bae64eda9aaf4ff5012fa95fR378
and handle that case exactly like previously but I don't think that we need to add that complexity. Let me know wdyt

Reply:
Its an interesting decision point because it's possible that fields may be initialized with some non-null value in their default constructors.  However, conceptually, I think what you are doing is the right thing here.  If the value is unset in C*, it should be effectively null in the entity itself.